packages/react/src/SectionIntro/SectionIntro.tsx

```diff
@@ -0,0 +1,79 @@
+import React, {forwardRef, type Ref} from 'react'
+import clsx from 'clsx'
+import {Button, type ButtonProps} from '../Button'
+import {Heading, HeadingTags} from '../Heading'
+import {Text} from '../Text'
+import styles from './SectionIntro.module.css'
+
+import type {BaseProps} from '../component-helpers'
+
+export type SectionIntroProps = BaseProps<HTMLDivElement> & {
+  children: React.ReactNode | React.ReactNode[]
+  align?: 'start' | 'center'
+}
+
+const Root = forwardRef<HTMLDivElement, SectionIntroProps>(({align = 'start', className, children, ...props}, ref) => {
+  return (
+    <div ref={ref} className={clsx(styles.SectionIntro, styles[`SectionIntro--align-${align}`], className)} {...props}>
```

Comment:
```suggestion
    <section ref={ref} className={clsx(styles.SectionIntro, styles[`SectionIntro--align-${align}`], className)} {...props}>
```

Reply:
I believe SectionIntro should not be a `section` itself but a descendant of a section. We might be able to use the `header` tag for it, though.

```
<section>
  <header class="SectionIntro">
    <h2>...<h2>
   </header>

   <!-- section content -->
</section>